.circleci/config.yml

```diff
@@ -332,7 +361,6 @@ jobs:
                 message: '<$CIRCLE_BUILD_URL|Release build for \`$VERSION\` started.>'
                 include_visit_job_action: true
             - make-deps
-            - run: pip3 install awscli
```

Comment:
This is needed for s3 uploads

Reply:
This has moved into the `make-deps` step.